src/FilterLocking/Components/FilterLockingInput/FilterLockingInput.js

```diff
@@ -106,6 +106,10 @@ export function FilterLockingInput(datamessenger, filterLocking){
             })
             return Promise.resolve(0)
         }else{
+			filterLocking.showSavingIndicator()
+			setTimeout(()=>{
+				filterLocking.hideSavingIndicator()
+			}, 1500);
```

Comment:
Instead of always calling showSavingIndicator then hideSavingIndicator after 1.5s, it might be better to make a function on filterLocking that does all that in one function. Something like function `displaySavingIndicatorFor (timeInMS) { }`

the custom timeInMS isn't necessary if it will never change